src/app/rulenode/config/components/action/send-email-config.tpl.html

```diff
@@ -61,6 +61,11 @@
         </md-input-container>
         <md-checkbox ng-disabled="$root.loading || readonly"
                      aria-label="{{ 'tb.rulenode.enable-tls' | translate }}" ng-model="configuration.enableTls">{{ 'tb.rulenode.enable-tls' | translate }}</md-checkbox>
+        <md-input-container class="md-block" ng-if="configuration.enableTls">
```

Comment:
This should be drop-down or something with automatic selection. User don't need to remember the syntax of TLS names

Reply:
done